content/docs/alerting/configuration.md

```diff
@@ -46,6 +46,7 @@ Generic placeholders are defined as follows:
 * `<secret>`: a regular string that is a secret, such as a password
 * `<tmpl_string>`: a string which is template-expanded before usage
 * `<tmpl_secret>`: a string which is template-expanded before usage that is a secret
+* `<uint64>`: an unsigned 64-bit integer value
```

Comment:
We use `int` over in the Prometheus config docs.

Reply:
Yeah, that's what I was wondering about. Should we really just use int everywhere in the docs, or should we also specify it more clearly in the main Prometheus docs? To make sure people don't set values that overflow or underflow.

Reply:
I'd say keep it simple with just int.

Reply:
Ok, changed to int.